### PR TITLE
PLT-3169 moving reload/recycle commands

### PR DESCRIPTION
**Pull request by coreyhulen**

### CODE REVIEW

File: webapp/components/admin_console/recycle_db.jsx [hunk 1]
```diff
@@ -44,6 +44,10 @@ export default class RecycleDbButton extends React.Component {
     }
 
     render() {
+        if (global.window.mm_license.IsLicensed !== 'true') {
+            return <div></div>;
+        }
+
```
Comment (enahum) [inline webapp/components/admin_console/recycle_db.jsx]: it seems that this line has trailing spaces

**Comment by jwilander** [comment]
+1, other than ESlint issue

**Comment by coreyhulen** [comment]
@enahum fixed eslint

**Comment by hmhealey** [comment]
Looks good. For future reference, I think you can make any tag self-closing when using JSX even if it doesn't let you do it normally in HTML

**Pull request merged**